.github/copilot-instructions.md

```
@@ -0,0 +1,537 @@
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <https://www.gnu.org/licenses/>.
+"""
+```
+
+**License**: GNU General Public License v3.0  
+**Copyright Holder**: Daniel Kampert  
+**Contact**: DanielKampert@kampis-elektroecke.de  
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)